logrotation-k8env/k8/deployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
```

Comment:
Please use "k8s". "k8" is incorrect term.

Reply:
sure 👍 , addressed.